dev/io.openliberty.data.internal_fat_jpa_hibernate/generated-src/test/jakarta/data/jpa/hibernate/integration/web/Point_.java

```diff
@@ -0,0 +1,32 @@
+package test.jakarta.data.jpa.hibernate.integration.web;
+
+import jakarta.annotation.Generated;
+import jakarta.persistence.metamodel.EmbeddableType;
+import jakarta.persistence.metamodel.SingularAttribute;
+import jakarta.persistence.metamodel.StaticMetamodel;
+
+@StaticMetamodel(Point.class)
+@Generated("org.hibernate.processor.HibernateProcessor")
+public abstract class Point_ {
```

Comment:
Can we delete this class?  It is probably left over from prior experimentation with entities.